### PR TITLE
Add `IsRelativeNameResolvableFromSymbol`

### PR DESCRIPTION
**Pull request by Krzysztof-Cieslak**
Continuation of https://github.com/fsharp/FSharp.Compiler.Service/pull/823

### CODE REVIEW

**Comment by vasily-kirichenko** [comment]
> D:\j\w\debug_windows---2f360ea4\src\fsharp\vs\service.fs(905,40): error FS0002: This function takes too many arguments, or is used in a context where a function is not expected

**Comment by Krzysztof-Cieslak** [comment]
@vasily-kirichenko, yes I've seen it. That's what happens if you send PRs from the train without working editor tooling ;-)

Will fix later today

File: src/fsharp/vs/service.fs [hunk 1]
```diff
@@ -900,6 +900,10 @@ type TypeCheckInfo
             (fun msg -> 
                 Trace.TraceInformation(sprintf "FCS: recovering from error in IsRelativeNameResolvable: '%s'" msg)
                 false)
+
+    /// Determines if a long ident is resolvable at a specific point.
+    member scope.IsRelativeNameResolvableFromSymbol(cursorPos: pos, plid: string list, symbol: FSharpSymbol) : bool =
```
Comment (dsyme) [inline src/fsharp/vs/service.fs]: Not the prettiest name but it will do :)

**Comment by dsyme** [comment]
I'm ok with this.  @vasily-kirichenko Do you think we need to add specific tests or is existing coverage of IsRelativeNameResolvable enough?

**Comment by vasily-kirichenko** [comment]
@dsyme we don't have any tests for `IsRelativeNameResolvable` :(

**Comment by Krzysztof-Cieslak** [comment]
If it's OK then I'd appreciate merge and getting FCS release with it 🙂

**Comment by dsyme** [comment]
@Krzysztof-Cieslak Could you add a test please under tests\service? thanks

**Comment by KevinRansom** [comment]
@dotnet-bot test this please

**Comment by Krzysztof-Cieslak** [comment]
@KevinRansom, thanks for merging.

@dsyme, I'll add tests in other PR, when I find some time to write it ;) I would be super happy if you could release FCS version with it, so we can use it in FSAC :-)

**Pull request merged**